### PR TITLE
Added List Component

### PR DESCRIPTION
**Pull request by anamkaa**
Added various types of list
- horizontal list
- vertical list
- bullet list
- numbered list

### CODE REVIEW

**Review by hrshmistry — COMMENTED** [review]
looks good, you can merge.

File: stylesheets/components/list.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    border-radius: 0.1rem;
+    padding: 0.5rem;
+    width: fit-content;
+}
+
+.list{
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+    margin: 0.5rem;
```
Comment (dev-enforced) [inline stylesheets/components/list.css:17]: You can create these properties as individual utility classes and apply it to the list item in the upcoming versions of it!!

File: stylesheets/components/list.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+.list-hr{
+flex-direction: row;
```
Comment (dev-enforced) [inline stylesheets/components/list.css:21]: flex-direction is by default row so you can avoid putting it

**Review by dev-enforced — APPROVED** [review]
The deploy preview of lists component looks good . Gave some minor suggestions to improve code quality in the upcoming versions
https://deploy-preview-39--fab-ui-lib.netlify.app/docs/docs-list.html
Good work @anamkaa

File: stylesheets/components/list.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/components/list.css:17]: Will implement it

File: stylesheets/components/list.css (hunk 2 above)
Comment (anamkaa) [reply stylesheets/components/list.css:21]: okay

**Pull request merged**